permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -86,41 +86,47 @@ class Permission {
   /// iOS: CoreMotion
   static const sensors = Permission._(12);
 
+  /// Android:
+  ///   When running on Android TIRAMISU and above: Background Sensors Permission
+  ///   When running on Android < TIRAMISU: Nothing
+  /// iOS: Nothing
+  static const sensorsAlways = Permission._(13);
```

Comment:
By renumbering all the enum values you introduce a breaking change in the interface. Please revert the renumbering and add a new entry for the `sensorsAlways` permission.